$NetBSD: patch-ah,v 1.3 2011/01/02 23:53:13 schnoebe Exp $

pkgsrc places idna.h in a different directory than other package
managers

--- iris/src/xmpp/xmpp-core/connector.cpp.orig	2012-10-04 00:53:09.008388608 +0000
+++ iris/src/xmpp/xmpp-core/connector.cpp
@@ -36,7 +36,7 @@
 #include <QUrl>
 #include <QTimer>
 #include <qca.h>
-#include <libidn/idna.h>
+#include <idna.h>
 
 #include "bsocket.h"
 #include "httpconnect.h"
